jsonpointer.py

```diff
@@ -309,12 +295,13 @@ def join(self, suffix):
             suffix_parts = suffix
         try:
             return JsonPointer.from_parts(chain(self.parts, suffix_parts))
-        except:
+        except:  # noqa E722
             raise JsonPointerException("Invalid suffix")
 
-    def __truediv__(self, suffix): # Python 3
+    def __truediv__(self, suffix):  # Python 3
         return self.join(suffix)
-    __div__ = __truediv__ # Python 2
+
+    __div__ = __truediv__  # Python 2
```

Comment:
Is that still needed when we no longer support Python 2.x?

Reply:
Good point, I'll remove